client/disputes/details/index.tsx

```diff
@@ -27,11 +27,19 @@ const DisputeDetails = ( {
 	query: { id: disputeId },
 }: {
 	query: { id: string };
-} ): JSX.Element => {
+} ): JSX.Element | null => {
 	const { dispute, isLoading, doAccept } = useDispute( disputeId );
 	const disputeObject = dispute || ( {} as Dispute );
 	const disputeIsAvailable = ! isLoading && dispute && disputeObject.id;
 
+	// Bail early and return nothing if the feature flag is not enabled.
+	// Here as a hint/reminder to delete this file when the feature flag is removed.
+	const isDisputeOnTransactionPageEnabled =
+		wcpaySettings.featureFlags.isDisputeOnTransactionPageEnabled;
+	if ( isDisputeOnTransactionPageEnabled ) {
+		return null;
+	}
+
```

Comment:
Added this so when we're working on #7289 we find this class (search for feature flag) and remember to delete it. 🐘

Reply:
I just added one check item in https://github.com/Automattic/woocommerce-payments/issues/7289:
> - [ ] Remove the [legacy dispute details page](https://github.com/Automattic/woocommerce-payments/blob/2eb4de5d4bca3573339e15197987a3943ffeddf4/client/disputes/details/index.tsx).